Gemfile

```diff
@@ -5,6 +5,8 @@ ruby '3.1.2'
 
 gem 'rubocop', '>= 1.0', '< 2.0'
 
+gem 'cancancan'
```

Comment:
Well, you made great effort for making this project. However, there are some errors that need to be fixed.

- [ ] Kindly make sure when you `Sign out` your page need to redirect you to login page.

Reply:
@Rustamxon7, please do you have any articles or resource that I can use to learn how to redirect to homepage with sign out button

Reply:
I think that might be helpful for you. [Link...](https://github.com/heartcombo/devise/issues/5458#:~:text=You%20have%20to,data%3A%20%7B%20turbo_method%3A%20%3Adelete%22%20%7D%20%25%3E)

Reply:
@Rustamxon7, my sign out works but I have to reload my page to redirect to the homepage